tests/test_tokenization_common.py

```diff
@@ -966,23 +976,43 @@ def test_internal_consistency(self):
 
     @require_tokenizers
     def test_encode_decode_with_spaces(self):
-        tokenizers = self.get_tokenizers(do_lower_case=False)
+        tokenizers = self.get_tokenizers(do_lower_case=False, fast=False)
         for tokenizer in tokenizers:
             with self.subTest(f"{tokenizer.__class__.__name__}"):
                 new_toks = [
-                    AddedToken("[ABC]", normalized=False),
-                    AddedToken("[DEF]", normalized=False),
-                    AddedToken("GHI IHG", normalized=False),
+                    AddedToken(
+                        "[ABC]", normalized=True, lstrip=True, rstrip=True
+                    ),  # these are added tokens, they will be normalized....
+                    AddedToken("[DEF]", normalized=True, lstrip=True, rstrip=True),
+                    AddedToken("GHI IHG", normalized=True, lstrip=True, rstrip=True),
                 ]
                 tokenizer.add_tokens(new_toks)
+                # special tokens should not be normalized...
+                tokenizer.add_tokens([AddedToken("[SAMPLE]", normalized=True)], special_tokens=True)
```

Comment:
Will users know to do this i.e. do we have docs / warnings?

Reply:
Actually this is no longer the case! You can just choose to normalize or not